addons/point_of_sale/static/src/js/models.js

```diff
@@ -2524,7 +2524,7 @@ class Order extends PosModel {
                 return paymentline.export_for_printing();
             });
         let partner = this.partner;
-        let cashier = this.pos.get_cashier();
+        let cashier = this.pos.get_order().cashier;
```

Comment:
```suggestion
        let cashier = this.cashier;
```

The method is already in the Order class, so take the cashier in the instance, not from the current order.

Reply:
Oops, Thanks for the review! 😄